int64 position 
---
int64 succes_bool
